src/main/resources/anniv6Resources/localization/eng/Hailstorm/UIstrings.json

```diff
@@ -0,0 +1,20 @@
+{
+  "${ModID}:Hailstorm": {
+    "TEXT": [
+      "Hailstorm",
+      "A #bnever #bending #bHail is taking over the spire. NL NL The Hailstorm caused campfires to die out, making it unable to rest. NL NL During combat, snow gives #yBlock to EVERYONE for the first #b3 turns, then hail deals damage to EVERYONE every turn. NL NL Encounter a new monster and a new event.",
+      "Diamsword"
+    ]
+  },
+  "${ModID}:FlintOption": {
+    "TEXT": [
+      "Rest twice",
+      "use the Flint to light a greater fire and rest twice."
+    ]
+  },
+  "${ModID}:HailEffect": {
+    "TEXT": [
+      "At the start of the first #b3 turns, EVERYONE gains #b4 #yBlock. NL After these #b3 turns, at the end of each turn EVERYONE takes #b4 damage, increased by #b1 every turn."
```

Comment:
At the end of each turn means twice every round?

Reply:
No, once per round.
I figured I'd use turn since "round" is not base game used, but I'll take any suggestions

Reply:
Maybe use at the end of the monster's turn then or something